src/main/java/hw_1/Task1.java

```diff
@@ -0,0 +1,13 @@
+package hw_1;
+
+public class Task1 {
+    public static void main(String[] args) {
+        String name = "Alina";
+        int age = 25;
+        float weight = 50.455f;
+
+        System.out.println(name);
+        System.out.println(age);
+        System.out.println(weight);
```

Comment:
Your output isn't as expected.
Please try to implement the same is in the task: Name: Alina, Age: 25, Weight: 50.45
![image](https://github.com/AntBelyi/ALevel_automation/assets/66726924/0a5466c6-c0d9-40cc-ba5a-a5923f847101)

So, the output should be in one line

Reply:
Done